metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/CostAggregator.java

```diff
@@ -0,0 +1,91 @@
+package io.dockstore.metricsaggregator.helper;
+
+import static io.dockstore.common.metrics.FormatCheckHelper.isValidCurrencyCode;
+import static io.dockstore.metricsaggregator.MoneyStatistics.CURRENCY;
+
+import io.dockstore.metricsaggregator.MoneyStatistics;
+import io.dockstore.openapi.client.model.Cost;
+import io.dockstore.openapi.client.model.CostMetric;
+import io.dockstore.openapi.client.model.Metrics;
+import io.dockstore.openapi.client.model.RunExecution;
+import io.dockstore.openapi.client.model.TaskExecutions;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import org.javamoney.moneta.Money;
+
+public class CostAggregator implements RunExecutionAggregator<CostMetric, Cost> {
+    @Override
+    public CostMetric getMetricFromMetrics(Metrics metrics) {
+        return metrics.getCost();
+    }
+
+    @Override
+    public Cost getMetricFromRunExecution(RunExecution runExecution) {
+        return runExecution.getCost();
+    }
+
+    @Override
+    public Optional<RunExecution> getWorkflowExecutionFromTaskExecutions(TaskExecutions taskExecutionsForOneWorkflowRun) {
```

Comment:
This is the new function that aggregates tasks into a workflow execution for the cost metric